src/Api/HttpClient.php

```diff
@@ -144,7 +144,7 @@ private function request($method, $path, array $params)
 
         // append request parameters to the URL
         if ('GET' === $method || 'DELETE' === $method) {
-            $path .= '?'.http_build_query($params);
+            $path .= '?'.http_build_query($params, null, '&');
```

Comment:
I guess the second parameter should be the empty string (see https://3v4l.org/gGkfE), shouldn't it?